src/data/set/finite.lean

```diff
@@ -828,6 +828,13 @@ finite.induction_on H
   (by simp only [bUnion_empty, bdd_above_empty, ball_empty_iff])
   (λ a s ha _ hs, by simp only [bUnion_insert, ball_insert_iff, bdd_above_union, hs])
 
+lemma infinite_of_not_bdd_above [_root_.infinite α] : ¬ bdd_above s → s.infinite :=
```

Comment:
I think these only need `nonempty α` as opposed to `infinite`

Reply:
Oh, lol, thanks -- that assumption wasn't even used!